src/reflect/value.go

```diff
@@ -187,7 +187,7 @@ func (v Value) Bool() bool {
 			return uintptr(v.value) != 0
 		}
 	default:
-		panic(&ValueError{"Bool"})
+			panic(&ValueError{Method: "Bool"})
```

Comment:
Not sure how all the spacing got messed up here. Happy to fix this up if this is something you'd like to merge.